src/styles/index.scss

```
@@ -1,3 +1,68 @@
+@import './utils';
+
 body {
   margin: 0;
+  padding: 0;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+
+    position: absolute;
+    top: 50%;
+    left: 50.5%;
+    transform: translate(-50%, -50%);
+
+    font-family: $day-font;
+    font-size: 30px;
+  }
+}
+
+.calendar {
+  width: 707px;
+  padding-inline: 10px;
+
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1px;
+
+  @each $day, $margin in $start-day-margins {
+    &--start-day-#{$day} {
+      .calendar__day:first-of-type {
+        margin-left: $margin;
+      }
+    }
+  }
+
+  @for $length from 28 through 31 {
+    &--month-length-#{$length} .calendar__day:nth-child(n + #{$length + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    width: $day-width;
+    height: $day-height;
+    position: relative;
+    box-sizing: border-box;
+
+    padding: 0 !important;
+
+    background-color: $day-color;
+    border: 1px black solid;
+    transition: transform 0.5s;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: $hover-color;
+    transform: translateY(-20px);
+  }
```

Use scss nesting for hover
```suggestion
    border: 1px black solid;
    transition: transform 0.5s;
    
    &:hover {
      // write styles here
    }
  }

```